threedi_modelchecker/exporters.py

```diff
@@ -53,6 +62,7 @@ def generate_rst_table(checks) -> str:
         + "     - Check message"
     )
     rst_table_string += header
+    checks = order_checks(checks)
     for check in checks:
```

Comment:
More a style suggestion. I'd not use an assignment in this case because it just adds unnecessary clutter.
```suggestion
    for check in order_checks(checks):
```